tests/mapper/test_cluster_gcn_node_mapper.py

```diff
@@ -0,0 +1,199 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2018-2019 Data61, CSIRO
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from stellargraph.mapper import ClusterNodeGenerator, ClusterNodeSequence
+from stellargraph.core.graph import StellarGraph
+
+import networkx as nx
+import pandas as pd
+import numpy as np
+import pytest
+
+
+def create_graph_features():
+    G = nx.Graph()
+    G.add_nodes_from(["a", "b", "c", "d"])
+    G.add_edges_from([("a", "b"), ("b", "c"), ("a", "c"), ("b", "d")])
+    G = G.to_undirected()
```

Comment:
My understanding is that `nx.Graph` is already undirected.

Reply:
You are correct. I've removed the extra call.